client/js/lounge.js

```diff
@@ -918,6 +924,23 @@ $(function() {
 		location.reload();
 	}
 
+	function updateDesktopNotificationStatus(){
+		var checkbox = $("#desktopNotifications");
+		var warning = $("#warnDisabledDesktopNotifications");
+
+		if (Notification.permission === "denied"){
+			checkbox.attr("disabled", true);
+			checkbox.attr("checked", false);
```

Comment:
I don't think it would be really usefull if the permission state is `default`. Because in the end, the user didn't care if it's on `default`. If he checks the box to activate the setting, he'll have the permission popup and it's done.